src/pages/migrating/guide/develop.mdx

```diff
@@ -79,199 +69,203 @@ To get started, uninstall the following packages if they exist in your project:
+<Row>
+<Column colLg={12}>
 
-**Note:** if you see any references to `@carbon/styles` for style imports, you
-can safely replace the beginning of the path with `@carbon/react` for the same
-import in this package.
+<Tabs>
 
-### Step 5: Update Sass imports
+<Tab label="npm">
 
-If your project is currently importing from different parts of
-`carbon-components` for specific variables, mixins, or functions then you will
-need to make sure that your imports have been updated to the correct path.
+```bash
+npm install sass --save-dev
+```
 
-The full list of changes are available in our
-[Migration Docs](https://github.com/carbon-design-system/carbon/blob/main/docs/migration/v11.md#changing-import-paths-from-scssglobalsscss-to-scss)
-but, in general, most imports have been updates so that the `carbon--` is no
-longer necessary.
+</Tab>
 
-### Step 6: Update JavaScript imports
+<Tab label="yarn">
 
-<InlineNotification>
+```bash
+yarn add sass --dev
+```
 
-This step can be automated using the `@carbon/upgrade` CLI. For full details,
-see the
-[`@carbon/upgrade`](https://github.com/carbon-design-system/carbon/tree/main/packages/upgrade)
-package documentation
+</Tab>
 
-</InlineNotification>
+</Tabs>
 
-**Rewrite imports from** `carbon-components-react` **to** `@carbon/react`.
+</Column>
+</Row>
 
-```bash
-npx @carbon/upgrade migrate update-carbon-components-react-import-to-scoped
-```
+Once you have Dart Sass installed, it's important that you configure your
+project to support resolving imports in Sass from `node_modules`. Typically,
+this means adding `node_modules` to your `includePaths` config for Sass in your
+bundler or toolchain of choice.
 
-After updating how you bring in styles, you will need to update your import
-paths from `carbon-components-react` to `@carbon/react` for components that
-you're bringing from Carbon.
+To learn more about how to configure your specific toolchain to support this,
+read the documentation for configuration
+[here](https://github.com/carbon-design-system/carbon/blob/main/packages/styles/docs/sass.md#configuration).
+We also have published a guide over on
+[Medium](https://medium.com/carbondesign/migrating-from-node-sass-to-sass-eba9db004a3a)
+to help out with common problems that come from this migration.
 
-```jsx
-// Before
-import { Accordion, AccordionItem } from 'carbon-components-react';
-// After
-import { Accordion, AccordionItem } from '@carbon/react';
-```
+### Step 3: Update style import paths
 
-### Step 7: Update icon imports
+Now, open your project's main stylesheet:
 
-<InlineNotification>
+- Any line of code with `@import` can be removed.
+- Any feature flags can be removed.
 
-This step can be automated using the `@carbon/upgrade` CLI. For full details,
-see the
-[`@carbon/upgrade`](https://github.com/carbon-design-system/carbon/tree/main/packages/upgrade)
-package documentation
+All you need is the following:
 
-</InlineNotification>
+```scss
+@use '@carbon/react';
+```
```

Comment:
This isn't true, there's lots of reasons why an import would need to remain (local stylesheets, user doesn't want to buy into sass module system yet, etc)

Also for some usages it's fine to just include everything via `@use '@carbon/react';` but we've seen a good majority of users opt out of this to instead include individual component styles to reduce their bundle size.